feature/qos/ate_tests/qos_output_queue_counters_test/qos_output_queue_counters_test.go

```diff
@@ -310,6 +310,11 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 		}
 	}
 
+	nc1InputWeight := uint64(200)
+	if deviations.SchedulerInputWeightLimit(ondatra.DUT(t, "dut")) {
+		nc1InputWeight = uint64(100)
```

Comment:
I don't see where SchedulerInputWeightLimit is documented to mean "use an input weight of 100"

Reply:
Please refer https://github.com/openconfig/featureprofiles/pull/1350 where the description of deviation is updated.

Reply:
Is this okay for NC1 and AF4 to have weight 100?